client/src/components/signUpForm/SignUpForm.js

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { useForm } from "react-hook-form";
+import SignUpButton from "../signUpButton/SignUpButton";
+import postAccount from "../../apiAccount/postAccount";
+const SignUpForm = () => {
+  const { register, handleSubmit, errors } = useForm();
+
+  const onSubmit = async (data) => {
```

Comment:
You could specify the data handled in this step for a better code understanding
```suggestion
  const onSubmit = async (userData) => {
```